### PR TITLE
feat: gh actions; improved deployment & update to latest version

### LINKED ISSUE #1: Review
@foosinn, @stp-ip, please review! :)

### LINKED ISSUE #8: Instructions on updating to 1.7.1

### PR DESCRIPTION
**Pull request by GoliathLabs**
Hi all,
I've been working on getting the latest strichliste working with the docker deployment. This is the result. I'm currently short on time, nonetheless I didn't want to forget about opening a PR. If everything looks good, feel free to merge this PR with squash mode on as there are quite a few commits in here.

Closes #8 #7

Maybe Closes #1 ? (Not sure what that's about)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,47 +7,47 @@ RUN apk --no-cache add ca-certificates \
...
+    php81-ctype \
+    php81-tokenizer \
+    php81-iconv \
+    php81-mbstring \
+    php81-xml \
+    php81-json \
+    php81-dom \
+    php81-pdo_mysql \
+    php81-fpm \
+    php81-session \
```
Comment (ccoors) [inline Dockerfile:30]: Could you also add `php81-sqlite3` and `php81-pdo_sqlite` so that `php bin/console app:import ...` works?

**Review by ccoors — APPROVED** [review]
Maybe replace `docker-compose` in the `README.md` with `docker compose`. LGTM otherwise, but I'm not a maintainer :)

**Comment by schinken** [comment]
I hope it was ready? :D

**Comment by schinken** [comment]
For me it looked fine, but I'm not an 100% expert on docker

**Comment by GoliathLabs** [comment]
Thank you. Can you also close issue #7, as it has been dealt with in this PR

**Pull request merged**